README.md

```diff
@@ -1,3 +1,90 @@
 # Filecoin Mechanical Twin
 
-Mechanistic model for the Filecoin Economy
+Mechanistic model for the Filecoin Economy. You can use this model to forecast all the components underlying Filecoin's circulating supply (i.e., minting, vesting, locking and burning), based on a set of parameters that encode storage provider behavior. The model uses the following assumptions:
+
+* Forecasting is done daily. This means that each forecasting step corresponds to a day and the forecasted metrics correspond to the value we expect to see at the end of that day.
+* The model uses the current sector states (i.e. known schedule expirations) and it estimates future onboardings and future renewals.
+* The daily power onboarded is a constant provided as a tunable parameter.
```

Comment:
We've upgraded this to also accept vector inputs which must be of the same length as the forecasting window.

Reply:
Good point! And the same for onboarded power and fil+ rate. I will fix the wording